server/lib/nl/fulfillment/correlation.py

```diff
@@ -93,48 +102,91 @@ def _populate_correlation_for_place(state: PopulateState, place: Place) -> bool:
     # Counter updated in get_sample_child_places
     return False
 
+  if utils.is_multi_sv(state.uttr):
+    lhs_svs, rhs_svs = _handle_multi_sv_in_uttr(state.uttr, places_to_check)
+  else:
+    lhs_svs, rhs_svs = _handle_svs_from_context(state.uttr, places_to_check)
```

Comment:
Clarifying question: so we give preference to multi_sv case over the context-based case when it comes to correlations? Any intuition for why that is the preference order?

Reply:
`is_multi_sv` being true is already a strong indication of the presence of multiple SVs (because that suggests the aggregate score of multi-sv exceeds the best single-sv).  but we are being conservative in acting on such a higher multi-sv score, by also requiring a comparison/correlation.

I'll add a note to refer to comments in `_route_comparison_or_correlation` where there's a discussion of this.